beacon-chain/rpc/prysm/v1alpha1/validator/proposer_attestations_electra.go

```diff
@@ -74,7 +74,7 @@ func computeOnChainAggregate(aggregates []ethpb.Att) ([]ethpb.Att, error) {
 			if err != nil {
 				return nil, err
 			}
-			committeeIndices[i] = helpers.CommitteeIndices(a.CommitteeBitsVal())[0]
+			cb.SetBitAt(uint64(helpers.CommitteeIndices(a.CommitteeBitsVal())[0]), true)
```

Comment:
```suggestion
			cb.SetBitAt(uint64(a.GetCommitteeIndex()), true)
```
This should do the trick too